undefined_syms.txt

```diff
@@ -9,3 +9,15 @@ D_0F000000 = 0x0F000000;
 
 // z_bg_mjin
 D_06000000 = 0x06000000;
+
+#if PLATFORM_N64
+func_801C7740_unknown = 0x801C7740;
+B_80121AE0 = 0x80121AE0;
+B_80121AE1 = 0x80121AE1;
+D_00B8ADA0 = 0x00B8ADA0;
+D_00B9DA70 = 0x00B9DA70;
+D_801DA410 = 0x801DA410;
+D_801E8090 = 0x801E8090;
+B_80121AE2 = 0x80121AE2;
+D_800FF4B0 = 0x800FF4B0;
```

Comment:
(Sorry, missed this earlier)
Some of these don't need to be undefined symbols:

0x800FF4B0: in code_n64dd_800AD4C0 .data
0x80121AE0: in src/code/code_n64dd_800AD410 .bss
0x80121AE1: in src/code/code_n64dd_800AD410 .bss
0x80121AE2: in src/code/code_n64dd_800AD410 .bss

Of course these file splits were invented by me so not sure which code_n64dd file they should go in. But the point is I think we should define them now